### DISCUSSION TITLE
Engaging new players until Land Expansions and beyond.

### DISCUSSION THREAD

**Discussion by Danverse98** [sunflower-land/sunflower-land#1176] [Ideas]
Hello farmers, from time to time i hear new players feeling left out because of the overall lack of NFTs they can look forward too. Many are sold out and players would like to experience the fun, minting part without investing sums of money and just feel the overall gameplay and mechanics before heading straight into investing into the more expensive NFTs. **Tl:dr** many new players have the feeling like there's not much to accomplish just by engaging in the sheer gameplay alone and that there's not much to do without investing. This suggestion is trying to help those issues, bringing engagement into the current state of the game and Land Expansions by serving mainly new and current players, with small opportunities to get deeper into the exciting feel of minting and collecting.

**Suggestion: a handful amount of tiny, mainly decorative NFTs for players starting out which will be _soul-bound without a limited supply_ so that the new players wouldn't head straight into fomo and enjoy the process of gathering and minting while figuring out the games potential, thus making the game a better place for casual players that are searching for a fun game to play.**

- I would call them **''Trophy'' NFTs** for now, since they will be soul-bound, their purpose is to learn/reward gameplay.

- Trophy NFTs should be oriented to **every aspect of the game** we currently have, from Sunflowers to Wheat, Wood to Gold and Chicken to Eggs, thus making the new players feel rewarded for learning every crop and material along their journey.

- I'd love the community to gather ideas on possible form of some Trophy NFTs, **they should bring engagement for new players and keep them on track, occupied while learning and playing the game with small rewards in return**(for example unlocking small Chicken oriented NFT after buying the first Chicken or Wheat oriented NFT after baking the Radish Pie; unlocking decorations after some levels or time spent in the game).

- Adding Achievements with some of them unlocking Trophy NFTs, adding more engagement for all players.

- Trophy NFTs could be **EXP oriented in Land Expansions**, rewarding players for learning the mechanics of the game.

- These gameplay mechanics fall nicely into place with the upcoming Farmer NFTs, with it's style oriented, uniqueness type of gameplay.

**Just so it's easier to imagine, Trophy NFTs are sort of like the mechanic of free skins and achievements other games implement. The idea of making a player feel like part of the community by gameplay alone and sticking around for a long time, while making them excited for the possibility to save up for his own unique skin which he can buy later.**

Up until Land Expansions, this will make the new players occupied and excited for the Land Expansions. After Land Expansions, this could also serve as somewhat of a guidance through the game and minting, getting people excited by building up their collection.

**I appreciate all the feedback, good or bad. If someone would like to bring this idea further with me, I'm all for it to make this idea in-depth surrounded with every detail so it fits the game, this is just a test to see the likeliness of these ideas. Thanks farmers for your time, take care. :)**

**Comment by adamhannigan** [comment]
Some great feedback! ❤️ We definitely will explore some more craftable items and release more content before land expansion.

Would these trophies still be available after land expansions?

**Reply by celpower** [reply]
I would hope that if something crafted is not going to be available in game after land expansion that it would be clarified before release. I would personally not spend money/resources on something only for beta. That would probably hurt even more the beginners that joined the game. Is that a risk for the NFTs crafted so far?

**Reply by Danverse98** [reply]
Thank you for your read and time! ❤️

Overall, this system could be a part of Beta, Land Expansions and beyond. In the current Beta stage, we could get a small form of this system which suits Beta and Land Expasions simultaneously, only because the game mechanics of Land Expansions will be different.

**_From what i know, the seed progress and raw materials should remain the same for now_, thus enabling us to come up with soul-bound NFTs that fit Beta and Land Expansions and adding more rewards when the gameplay itself branches out.**

- **Wheat NFT** after harvesting your first few batch of Wheat.
- **Mineral NFT** after mining all of the possible minerals.
- **Chicken oriented NFT** after harvesting aprrox. 30 eggs.
- **Rookie collector NFT** for owning more than 3 NFTs(not soul-bound ones).
- **Lvl 10** Farming and Gathering NFTs.
- an NFT for **being part of this game** for more than 2-3 weeks.

- The overal appearance of those tiny, Trophy NFTs **could be whatever** honestly, leaving a free hand to the artists to come up with as many tiny decorations as possible we can then implement. For example, by achieving enough to get the Wheat oriented Trophy, the design of the Trophy may not even have to be in the looks of a Wheat. It's just the fact that a player got a decoration they will **cherish and be proud of, reminding them of their achievements they made.**

**They can be crafted at the Blacksmith's Craft for now**, changed with Land Expansions after, noticing you with a pop-up(same as news do with the recent update) that you unlocked a small reward.

- **These NFTs should be tiny**, nothing big that floods the town or the future land. We don't want to give the players a flashy collection, the main focus is that they should feel rewarded for their gameplay long-term.

**With Land Expansion, i see that the overall gameplay will be more advanced, thus giving the opportunity for more Trophy NFTs to be added, going along with the gameplay and progress.**

Again, this system should **higher the chances of players sticking around for a long time**, negating newcomers of the idea that ''i'm too late' or what's my focus now'' and making them engaged. This system could get more advanced and difficult by time with more Trophy NFTs coming out, thus engaging the long term players even more.

I see this system as **not being part of** the overall NFT economy, being independent from values and only focusing on the players engagement and the feeling of being rewarded for the gameplay and time a players puts into the game.

**Comment by Rudolph-the-Bumpkin** [comment]
Hey @adamhannigan! Was revisiting some old proposals and would be great to replace the current achievement rewards for different trophies :)

**Reply by brian-man** [reply]
> Hey @adamhannigan! Was revisiting some old proposals and would be great to replace the current achievement rewards for different trophies :)

Some of us are still chasing the current achievements.  For the current achievements that have collectible rewards, please don't replace or remove them.
